packages/wallet/src/index.ts

```diff
@@ -1,59 +1,58 @@
-/* eslint-disable ethereumjs/noBuffer */
 import {
   bytesToHex,
+  bytesToPrefixedHexString,
   importPublic,
   isValidPrivate,
   isValidPublic,
   privateToAddress,
   privateToPublic,
   publicToAddress,
+  randomBytes,
   toChecksumAddress,
 } from '@ethereumjs/util'
 import { base58check } from '@scure/base'
 import * as aes from 'ethereum-cryptography/aes'
 import { keccak256 } from 'ethereum-cryptography/keccak'
 import { pbkdf2 } from 'ethereum-cryptography/pbkdf2'
-import { getRandomBytesSync } from 'ethereum-cryptography/random'
 import { scrypt } from 'ethereum-cryptography/scrypt'
 import { sha256 } from 'ethereum-cryptography/sha256'
+import { concatBytes, equalsBytes, hexToBytes, utf8ToBytes } from 'ethereum-cryptography/utils'
 
-import { bufferToHex } from './util/bytes'
+import type { PrefixedHexString } from '@ethereumjs/util'
 
 export { EthereumHDKey as hdkey } from './hdkey'
 export { Thirdparty as thirdparty } from './thirdparty'
 const uuidv4 = require('uuid').v4
 
 const bs58check = base58check(sha256)
-function randomBytes(num: number) {
-  return Buffer.from(getRandomBytesSync(num))
-}
+
 interface KDFParamsV1 {
   N: number
   P: number
   R: number
   DkLen: number
 }
-function scryptV1(password: Buffer, salt: Buffer, kdfparams: KDFParamsV1) {
+function scryptV1(password: Uint8Array, salt: Uint8Array, kdfparams: KDFParamsV1) {
   const { N, P, R, DkLen } = kdfparams
   return scrypt(password, salt, N, P, R, DkLen)
 }
 function scryptV3(password: string, kdfparams: ScryptKDFParams) {
   const { salt, n, p, r, dklen } = kdfparams
-  return scrypt(Buffer.from(password), salt, n, p, r, dklen)
+  return scrypt(utf8ToBytes(password), salt, n, p, r, dklen)
 }
 function scryptV3Out(password: string, kdfparams: ScryptKDFParamsOut) {
   const { salt, n, p, r, dklen } = kdfparams
-  return scrypt(Buffer.from(password), Buffer.from(salt, 'hex'), n, p, r, dklen)
+  return scrypt(utf8ToBytes(password), hexToBytes(salt), n, p, r, dklen)
```

Comment:
Better ask than sorry: why utf8ToBytes for password and hexToBytes for salt? (can't look up, on mobile, but both are string converters I assume?

Reply:
Because the `password` can be any string, not just a hex string.  This is anything the user provides to encrypt/decrypt a key (like decrypting/encrypting an SSH key)